parsl/dataflow/strategy.py

```diff
@@ -262,7 +262,10 @@ def _general_strategy(self, status_list, tasks, *, strategy_type):
                     logger.debug("More slots than tasks")
                     if isinstance(executor, HighThroughputExecutor):
                         if active_blocks > min_blocks:
-                            exec_status.scale_in(1, force=False, max_idletime=self.max_idletime)
+                            exec_status.scale_in(
+                                active_blocks - active_tasks // tasks_per_node // nodes_per_block,
```

Comment:
I think this calculation should look similar to the calculation around line 246: both of them arg calculating some kind of "target number of blocks" and it makes me uncomfortable that they don't look exactly the same (eg `ceil`, `min` and `parallelism`)

other than that, this looks like the right thing to be doing.